### ISSUE TITLE
virtual keyboard not activated on tablet when trying to enter a letter in the crossword

### ISSUE THREAD

**Issue by conseildepartemental85** [geraldloss/glcrossword#3]
Hi there, GLcrossword is a great game for our Typo3 site.

The downside though is that it is not playable on tablet at the moment...

I have tried to force the virtual keyboard to popup when you try to enter a Letter in the crossword but it doesn't work

Would you have a solution to this problem so that it would be possible to play crossword on tablet ?

Thanks for your reply

**Comment by geraldloss** [comment]
Hi hi, I have checked this issue, but this seams not that easy. I have found no native function in Javascript or JQuery to show up the keyboard on iOS or Android. The only way I found, I need to to place an inputbox in every square of the crossword. That means I need major changings in the html structure of the crossword. Thats not so easy to do without changing the look and behaviour of the crossword. And maybe many parts of the javascript code which now rely to find the text in a <div> has to change to an <input>.
If I find some more time I will try to fix this. I'm interested as well to provide mobile support, but having much time is currently an issue :-(

Regards
Gerald

**Comment by conseildepartemental85** [comment]
Hi, I understand what you mean.  We have been trying to fiddle around with the JS and html. we have managed to make the keyboard appear on IOS but it's not working properly though. As you said inputboxes need to be set (or replace the div) of every square. As for Android it shows the keyboard but allows more than oneletter per box input... making the JS function not working.

ATM we have desactivated the game on mobile and tablets with a replacement text for users to indicate that the game is only made for desktop.

Thanks for your reply and work.

Hope to see a responsive version of your nice game at one point ^^

regards.
